docs/using/event-types.md

```diff
@@ -68,6 +74,9 @@ The event type has a simple JSON Schema submitted as an escaped JSON string desc
 says events will be randomly allocated to partitions, and the owner's name is
 `"acme-order-service"`. The `enrichment_strategies` array says to apply `metadata_enrichment` to submitted events (common metadata is a feature of some categories).
 
+The event type has an optional `default_statistics` object, which controls the number of partitions. Nakadi will use a sensible default if no value is provided. The values provided here 
```

Comment:
because of my mistake the field is called `default_sttistic`, not `default_statistics`